graphQl/resolvers.js

```diff
@@ -1,28 +1,30 @@
-const bcrypt = require("bcrypt");
-const validator = require("validator");
-const jwt = require("jsonwebtoken");
-const User = require("../models/User");
+const bcrypt = require('bcrypt');
+const validator = require('validator');
+const jwt = require('jsonwebtoken');
+const User = require('../models/User');
+
 module.exports = {
+  // eslint-disable-next-line no-unused-vars
   async signUp({ UserInput }, req) {
     const errors = [];
     if (!validator.isEmail(UserInput.email)) {
-      errors.push({ message: " Email is invalid" });
+      errors.push({ message: ' Email is invalid' });
     }
     if (
-      validator.isEmpty(UserInput.password) ||
-      !validator.isLength(UserInput.password, { min: 5 })
+      validator.isEmpty(UserInput.password)
+      || !validator.isLength(UserInput.password, { min: 5 })
```

Comment:
```suggestion
      validator.isPasswordValid(UserInput.password);
```
This function should take care of everything.